test_list.py

```diff
@@ -0,0 +1,35 @@
+
+TESTS = [
+    'black-page',
+    'white-page',
+    'red-page'
+    #'idle-foreground',
+    #'idle-background'
+]
+
+TEST_COMMANDS = {
+	'black-page': [
+		"adb",
+		"shell",
+		"am start -n org.mozilla.firefox/.App " \
+		"-a android.intent.action.VIEW " \
+		"""-d "data:text/html;base64,PGJvZHkgc3R5bGU9ImJhY2tncm91bmQtY29sb3I6YmxhY2siPg==" """ \
+		"--ez showstartpane false"
+	],
+	'white-page': [
+		"adb",
+		"shell",
+		"""am start -n org.mozilla.firefox/.App """ \
```

Comment:
So this is different from

```
"am start -n org.mozilla.firefox/.App " \
```

which ```black-page``` and ```red-page``` use, but I don't necessarily have an issue with that; if we can (and I think we can in all these initial tests), we should be consistent.

Reply:
(I'll try to dig up some pointers/conversation with @ncalexan (whom I just GitHub-name-dropped, here; hello!) about the various desired/optimal grouping and use.

Reply:
There's no reason to not be consistent.  And in fact, you don't _need_ `.App`, since you didn't end up wanting `file:///` URLs (which might have shown a difference, for historical reasons).

Also, I don't see a difference between the invocations.  But why wouldn't you generate the base64 strings in your harness?  Future you will appreciate a) knowing what the underlying code is and b) being able to update it without frustration.

Reply:
i.e., `def command_for_color(...): -> [...]` with the color name baked in or whatever.  DRY!

Reply:
It's been changed to use a `command_for_color` function. Now, we pass a color to `--color` to set the color of the page in testing. I've also made another change to move the pre_post script to a general script and made a specific script for color tests. We should make the pre_post functionality more general in a follow-up PR.